README.md

```diff
@@ -1 +1,99 @@
-Coming soon…
+# UI Component Best Practices <!-- omit in toc -->
+
+A bunch of best practices to help achieve User Interface (UI) components, or
+presentational components, that are:
+
+- accessible
+- consumable
+- portable
+- reusable
+- and robust.
```

Comment:
As this is a bullet list, could probably just say `robust` without `and` and the `.`

Reply:
I was using the list in sentence form. It was missing the commas after the list items bar the last one. And I formatted "and" a little differently:

```
- accessible,
- consumable,
- portable,
- reusable, and
- robust.
```